### ISSUE TITLE
ePub book content overlap lines and are unreadable

### ISSUE THREAD

**Issue by fabiocolacio** [babluboy/bookworm#210]
Books in the epub format (I have not tested other formats) are unreadable in Fedora 28. 

These screenshots are taken in Fedora 28, with bookworm compiled from git-master:

The default view with no settings changed:

![screenshot from 2018-08-04 00-27-24](https://user-images.githubusercontent.com/5248352/43673762-32d0a1ca-977d-11e8-8ef8-8d2624627c0a.png)

With two-page reading enabled:

![screenshot from 2018-08-04 00-17-21](https://user-images.githubusercontent.com/5248352/43673698-1921fee6-977c-11e8-9399-ae4583132961.png)

**Comment by babluboy** [comment]
@fabiocolacio Thanks for your interest in Bookworm. On first thoughts this does not look like a distro issue. Although I have not seen this on eOS, I am thinking it’s specific to the ePub formatting which is making the rendering bad.

Any chance you can share the ePub file with me on this issue or on my email (bablu.boy@gmail.com) so that I can test it.

**Comment by fabiocolacio** [comment]
Hi @babluboy,

That seems like a probable cause of for this issue. All of the books I have tested come from [Project Gutenberg](https://www.gutenberg.org/wiki/Main_Page), and thus they are likely to be formatted in a similar manner. Every book I have tried from this site has had the same issues (I do not have any other epubs to test aside from these).

The particular file shown in the screenshots above is White Nights by Fyodor Dostoyevsky, which can be downloaded from Project Gutenberg [here](https://www.gutenberg.org/ebooks/36034) (choosing images or no-images made no difference in my testing).

I should add that the same files render properly in other epub readers I have tested including [libgepub](https://gitlab.gnome.org/GNOME/libgepub/), mupdf, and Calibre.
